assignments/array-methods.js

```diff
@@ -55,22 +55,52 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+
+
+
 let fullName = [];
+
+
+
+
+runners.forEach(runner => fullName.push(`${runner.first_name} ${runner.last_name}`));
+
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
+
 let allCaps = [];
-console.log(allCaps); 
 
+const allCap = runners.map( function(currentValue){
```

Comment:
remember you already defined `allCaps` with a let statement a few lines above so you don't need to do so again: now you can just assign it a new value with `allCap = runners.map(etc.)`